docs/architecture/adr-001b-configuration-management.md

```diff
@@ -0,0 +1,115 @@
+# Architecture Decision Record: Configuration Management
+
+## Context
+
+Configuration management for DPL CMS is a complex issue. The complexity stems
+from [different types of DPL CMS sites](../configuration-management.md).
+
+There are two approaches to the problem:
+
+1. All configuration is local unless explicitly marked as core configuration
+2. All configuration is core unless explicitly marked as local configuration
+
+A solution to configuration management must live up to the following test:
+
+1. Initialize a local environment to represent a site
+2. Import the provided configuration through site installation using
+   `drush site-install --existing-config -y`
+3. Log in to see that Core configuration is imported. This can be verified if
+   the site name is set to DPL CMS.
+4. Change a Core configuration value e.g. on <http://dpl-cms.docker/admin/config/development/performance>
+5. Run `drush config-import -y` and see that the change is rolled back and the
+   configuration value is back to default. This shows that Core configuration
+   will remain managed by the configuration system.
+6. Change a local configuration value like the site name on <http://dpl-cms.docker/admin/config/system/site-information>
+7. Run `drush config-import -y` to see that no configuration is imported. This
+   shows that local configuration which can be managed by Editor libraries will
+   be left unchanged.
+8. Enable and configure the Shortcut module and add a new Shortcut set.
+9. Run `drush config-import -y` to see that the module is not disabled and the
+   configuration remains unchanged. This shows that local configuration in the
+   form of new modules added by Webmaster libraries will be left unchanged.
+
+## Decision
+
+We use the
+[Configuration Ignore module](https://www.drupal.org/project/config_ignore)
+and the [Config Ignore Auto module](https://www.drupal.org/project/config_ignore_auto)
+to manage configuration.
+
+The base module maintains a list of patterns for configuration which will be
+ignored  during the configuration import process. This allows us to avoid
+updating local configuration.
+
+Here, we can add some of the settings files that we already know needs to be
+ignored and admin-respected.
+But in reality, we don't need to do this manually, because of the second module:
+
+**Config Ignore Auto** is only enabled on non-development sites.
```

Comment:
This sounds like we only install the module on these sites but that is not actually the case.

Instead we rely on the status/enabling the module which is part fot the settings.